src/quick_start.rs

```diff
@@ -6,7 +6,6 @@ use ndarray::Array;
 // nalgebra
```

Comment:
this is unrelated to this specific change, but could you change the comment of the line 121 of this file?